requirements.txt

```diff
@@ -1,6 +1,5 @@
 git+https://github.com/adsabs/ADSPipelineUtils.git@v1.2.1
-git+https://github.com/adsabs/adsabs-pyingest.git@v1.0.7
 alembic==0.9.1
 psycopg2==2.8.3
 Jinja2==2.10.1
-requests==2.19.1
\ No newline at end of file
+requests==2.19.1
```

Comment:
ADSPipelineUtils provides a standarized HTTP client (with connection pool) and it is used by this pipeline, it seems that the requests package is not needed here (it will be pulled by ADSPipelineUtils).

Reply:
Sounds good, hopefully I can figure it out!